content/_data/changelogs/lts.yml

```
@@ -3604,6 +3604,213 @@
+    - MRamonLeon
+    message: |-
+      Prevent Old Data Monitor from failing plugin loading in the case of class field unmarshalling issues.
+
+  lts_changes: # compared to lts_predecessor 2.222.4
+
+  # Plugin manager improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4513
+    issue: 61166
+    references:
+    - issue: 61166
+    - pull: 4580
+    - pull: 4513
+    - pull: 4588
+    - pull: 4534
+    - pull: 4591
+    - pull: 4535
+    - pull: 4589
+    - pull: 4584
+    authors:
+    - daniel-beck
+    message: |-
+      Don't show all available plugins by default; use search field to find plugins.
+      Indicate when security issues would be addressed by an update in plugin manager.
+      Sort plugins by popularity on the <em>Available</em> plugin manager tab if the update site provides popularity data.
+      Show plugin categories as labels in the plugin manager instead of grouping them into different table sections.
+      Allow use of multiple space-separated filter terms in plugin manager.
+      Allow linking to plugin manager URLs with pre-filled filter field.
+      Link labels in the plugin manager to pre-filtered lists.
+      Show plugin release date in plugin manager.
+      List plugins that failed to load on the Installed tab of the plugin manager.
+      Highlight in the plugin manager when plugins are looking for new maintainers ("Adopt this plugin").
+
+  # Major user experience improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4546
+    authors:
+    - daniel-beck
+    message: |-
+      Organize entries on the <em>Manage Jenkins</em> page into categories and show them in a grid.
+
+  # Other improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4503
+    authors:
+    - daniel-beck
+    message: |-
+      Remove 'auto refresh' feature, including now obsolete auto refresh telemetry capability.
+
+  # System/Read improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4479
+    issue: 61202
+    authors:
+    - timja
+    references:
+    - issue: 61202
+    - pull: 4479
+    - url: https://www.jenkins.io/blog/2020/05/25/read-only-jenkins-announcement/
+      title: Read-only Jenkins Configuration blog post
+    message: |-
+      Users with extended read permission now get a more read-only looking UI.
+  - type: major rfe
+    category: major rfe
+    pull: 4520
+    issue: 61201
+    references:
+    - issue: 61205
+    - pull: 4524
+    - issue: 61201
+    - pull: 4520
+    - issue: 61203
+    - pull: 4518
+    - issue: 61207
+    - pull: 4530
+    - issue: 61455
+    - pull: 4569
+    - issue: 61208
+    - pull: 4533
+    - issue: 61208
+    - pull: 4685
+    authors:
+    - timja
+    - amuniz
+    - daniel-beck
+    message: |-
+      Allow users with system read permission to view the <em>About Jenkins</em> page, the <em>Manage Plugins</em> configuration, the global security configuration page, the admin monitors, and the system logs.
```

The admin monitors one is of a different _kind_, it's not a specific feature but only adds the supporting infrastructure. I would leave that one separate and rephrase this one as follows for internal consistency:

```suggestion
      Allow users with Overall/SystemRead permission to view <em>About Jenkins</em>, <em>Manage Plugins</em>, <em>Global Security Configuration</em>, and <em>System Log</em>.
```